includes/Core/Authentication/Authentication.php

```diff
@@ -633,6 +648,15 @@ private function inline_js_setup_data( $data ) {
 			$data['isVerified'] = false;
 		}
 
+		add_filter(
+			'googlesitekit_user_data',
+			function( $user ) use ( $data ) {
+				$user['verified'] = $data['isVerified'];
+				return $user;
+			}
+		);
```

Comment:
Same here, let's move this to the `register` method and duplicate the relevant logic, since `inline_js_setup_data` is legacy code that this shouldn't be tied into.